projects/igniteui-angular/src/lib/directives/for-of/for_of.directive.ts

```diff
@@ -425,19 +425,23 @@ export class IgxForOfDirective<T> implements OnInit, OnChanges, DoCheck, OnDestr
         if (index < 0 || index > (this.isRemote ? this.totalItemCount : this.igxForOf.length)) {
             return;
         }
-        // this.state.startIndex = index;
+        const containerSize = parseInt(this.igxForContainerSize, 10);
+        const scr = this.igxForScrollOrientation === 'horizontal' ?
+            this.hScroll.scrollLeft : this.vh.instance.elementRef.nativeElement.scrollTop;
```

Comment:
This might not work when we are over the browser scroll height limitation because the actual scroll top would be
scrollbarScrollTop * virtualScrollHeightRatio.